src/plugins/data/public/actions/value_click_action.ts

```diff
@@ -26,21 +26,22 @@ import {
 } from '../../../../plugins/ui_actions/public';
 import { getOverlays, getIndexPatterns } from '../services';
 import { applyFiltersPopover } from '../ui/apply_filters';
-import { createFiltersFromEvent } from './filters/create_filters_from_event';
+import {
+  createFiltersFromValueClickAction,
+  ValueClickEvent,
+} from './filters/create_filters_from_value_click';
 import { Filter, FilterManager, TimefilterContract, esFilters } from '..';
 
 export const ACTION_VALUE_CLICK = 'ACTION_VALUE_CLICK';
 
 export interface ValueClickActionContext {
-  data: any;
+  data: ValueClickEvent;
   timeFieldName: string;
 }
 
 async function isCompatible(context: ValueClickActionContext) {
   try {
-    const filters: Filter[] =
-      (await createFiltersFromEvent(context.data.data || [context.data], context.data.negate)) ||
-      [];
+    const filters: Filter[] = await createFiltersFromValueClickAction(context.data);
```

Comment:
Looks like in original implementation there was a weird case covered by ` [context.data]` looks like in isn't covered by `createFiltersFromValueClickAction` implementation

Reply:
i tried to find any place that would be passing in context.data without the property .data and i can't find it